server/service/handler.go

```diff
@@ -383,8 +383,13 @@ func attachFleetAPIRoutes(r *mux.Router, svc fleet.Service, config config.FleetC
 	ue.PATCH("/api/_version_/fleet/software/titles/{title_id:[0-9]+}/app_store_app", updateAppStoreAppEndpoint, updateAppStoreAppRequest{})
 
 	// Setup Experience
+	//
+	// Setup experience software endpoints:
 	ue.PUT("/api/_version_/fleet/setup_experience/software", putSetupExperienceSoftware, putSetupExperienceSoftwareRequest{})
+	ue.PUT("/api/_version_/fleet/setup_experience/{platform}/software", putSetupExperienceSoftware, putSetupExperienceSoftwareRequest{})
 	ue.GET("/api/_version_/fleet/setup_experience/software", getSetupExperienceSoftware, getSetupExperienceSoftwareRequest{})
+	ue.GET("/api/_version_/fleet/setup_experience/{platform}/software", getSetupExperienceSoftware, getSetupExperienceSoftwareRequest{})
```

Comment:
The current macOS setup experience logic uses `.../fleet/setup_experience/software`. Will this be fully functional as well for `macos`? If so may be worthwhile to update that API call as well to the new path